app/src/main/java/com/mobile/tandil/javabaseproject/mvp/view/DialogView.java

```diff
@@ -0,0 +1,27 @@
+package com.mobile.tandil.javabaseproject.mvp.view;
+
+import android.app.DialogFragment;
+import android.widget.Toast;
+import com.mobile.tandil.javabaseproject.R;
+import com.mobile.tandil.javabaseproject.fragment.ParkingDialogFragment;
+import com.mobile.tandil.javabaseproject.listener.ListenerDialogFragment;
+import com.mobile.tandil.javabaseproject.mvp.contract.DialogFragmentContract;
+import com.mobile.tandil.javabaseproject.mvp.view.base.FragmentView;
+
+public class DialogView extends FragmentView implements DialogFragmentContract.View {
+
+    public DialogView(ParkingDialogFragment fragment) {
+        super(fragment);
+    }
+
+    @Override
+    public void showParkingAvailable(String parkingAvailable, ListenerDialogFragment dialogFragmentListener) {
+        DialogFragment fragment = (DialogFragment) getFragment();
+        dialogFragmentListener.listenAvailableNumber(parkingAvailable);
+        fragment.dismiss();
+    }
+
+    public void showInvalidMessage() {
+        Toast.makeText(getContext(), getContext().getString(R.string.error_message), Toast.LENGTH_LONG).show();
```

Comment:
every time you use a getContext() you should validate that it is not null. Please add that validation before using it

Reply:
Done